README.md

```diff
@@ -211,34 +221,69 @@ Link to other services:
 docker-compose -f docker-compose.yaml -f docker-compose.link.yaml up
 ```
 
+# How to get an output
+
+There are 4 possible outcomes depending on the input:
+
+1. The input is a valid payment file.
+Outcome:
+    - The transactions are logged,
+    - The payment file is moved from `inbound` to `archive`,
+    - Each valid payment request is sent to the Topic `ffc-pay-request`.
```

Comment:
What do you think about reference the topic names?  I think it's good to include them but in local and every environment, they have a suffix?  Should we reference there is a suffix, ie `ffc-pay-request-ENV` or change it to say begins with `ffc-pay-request`?